src/topology/locally_constant/algebra.lean

```diff
@@ -117,6 +117,14 @@ instance [non_unital_semiring Y] : non_unital_semiring (locally_constant X Y) :=
 instance [non_assoc_semiring Y] : non_assoc_semiring (locally_constant X Y) :=
 { .. locally_constant.mul_one_class, .. locally_constant.non_unital_non_assoc_semiring }
 
+/-- The constant-function embedding, as a ring hom.  -/
+@[simps] def const_ring_hom [non_assoc_semiring Y] : Y →+* locally_constant X Y :=
+{ to_fun    := const X,
+  map_one'  := rfl,
+  map_mul'  := λ _ _, rfl,
+  map_zero' := rfl,
+  map_add'  := λ _ _, rfl, }
+
```

Comment:
If you wanted you could provide this for `const_add_monoid_hom` too

Reply:
Done!